instance.go

```diff
@@ -13,16 +16,26 @@ type instance struct {
 	*ec2.Instance
 	name      string
 	privateIP net.IP
+        ipv6_addresses string
```

Comment:
minor question: why is this indented in?

Reply:
because my editor is stupid ;)

Reply:
👌 — that clarifies it 😎